library/alloc/src/boxed/thin.rs

```diff
@@ -69,6 +69,27 @@ impl<T> ThinBox<T> {
     }
 }
 
+#[unstable(feature = "thin_box", issue = "92791")]
+impl<T> ThinBox<T> {
```

Comment:
```suggestion

```

These impl blocks are identical. They should be combined into one block, otherwise rustdoc will render it as two different impl blocks.

<table><tr><td><img src="https://github.com/rust-lang/rust/assets/1940490/8b3730c8-17f9-4dc4-bbe4-11552cb3d989"></td><td><img src="https://github.com/rust-lang/rust/assets/1940490/236b9177-e305-424e-b3c5-3ab30bae6df4"></td></tr></table>